crates/nu-command/src/conversions/into/datetime.rs

```diff
@@ -575,6 +580,24 @@ mod tests {
         assert_eq!(actual, expected)
     }
 
+    #[test]
+    fn takes_int_with_formatstring() {
+        let date_int = Value::test_int(1_614_434_140);
+        let fmt_options = Some(DatetimeFormat("%s".to_string()));
+        let args = Arguments {
+            zone_options: None,
+            format_options: fmt_options,
+            cell_paths: None,
+        };
+        let actual = action(&date_int, &args, Span::test_data());
+        let expected = Value::date(
+            DateTime::parse_from_str("2021-02-27 21:55:40 +08:00", "%Y-%m-%d %H:%M:%S %z").unwrap(),
```

Comment:
It seems like a perfectly reasonable conversion to me, but double-check my logic to make sure:

* A Unix epoch is the number of seconds since 00:00:00 UTC on January 1, 1970 UTC - That timezone is explicit.
* That means that any number of seconds will be <x> time in UTC, but also <y> time in another timezone. So ...
  * 1614434140 is 2021-02-27 13:55:40 +0000
  * 1614434140 is also 2021-02-27 21:55:40 +08:00
  * 1614434140 is also 2021-02-27 19:55:40 +0600
* So, parsing any of the above through `parse_from_str` would result in the same Unix epoch 

Why `+0800` here?  No idea - I honestly just copied the value from the [existing test](https://github.com/nushell/nushell/blob/56e2ce8f73fdfdfbaa68d50a22b4c4ba56fc13fe/crates/nu-command/src/conversions/into/datetime.rs#L563) (dropping the nanoseconds to seconds, of course) since I knew it was already a valid conversion/test. ;-)

Reply:
ok, sounds good. thanks for explaining your logic.